Gh pages

andyavt opened the pull request.

ati committed:
add front matter to markdown doc

andyavt committed:
How to implement a GPS trackers connection

andyavt committed:
fixed line under number

andyavt committed:
fixed white space

andyavt committed:
How to route the SMS messages through an external gateway

andyavt committed:
New Docs style

andyavt committed:
removed question mark

andyavt committed:
Fixed mistake in picture

There was VSAT tREminal

andyavt committed:
push grammar changes

andyavt committed:
Setting marks

andyavt committed:
Bug testing

andyavt committed:
2.1 Antennas installation

andyavt committed:
Removed dots

andyavt committed:
Handover + log level published

andyavt committed:
Pics change at  2.1

andyavt committed:
Publish fix #1

andyavt committed:
Checking one feature

andyavt committed:
New Theory + test of page proof

andyavt committed:
h2 test

andyavt committed:
'Note' check

andyavt committed:
Font problem testing

andyavt committed:
Revert "Font problem testing"

This reverts commit 597e6a8372c6499581ec26a22fd61f4bb85bad63.

andyavt committed:
font problem 'strong'

andyavt committed:
Theory update + grammar check. Published

andyavt committed:
<strong> changes

andyavt committed:
Test#1

andyavt committed:
Revert "Test#1"

This reverts commit 89f60dec5c0717feda6a99054f0de12a95923816.

andyavt committed:
Revert "Theory update + grammar check. Published"

This reverts commit 1f7ea83ad652cd329538fe358113a72c815626bf.

Conflicts:
	soft_conf.html

andyavt committed:
Revert back

andyavt committed:
mistakes fixed

andyavt committed:
Move docs to Um-docs

andyavt committed:
UHD Driver updates. Published

andyavt committed:
Signs fix

andyavt committed:
text formatting

andyavt committed:
UmDESK text formatting

andyavt committed:
GSM parameters.

andyavt committed:
Adding 'Refer'

andyavt committed:
Antennas installation. Draft#2

andyavt committed:
21.07 Updates publishing

andyavt committed:
Minor Fix

andyavt committed:
GPS Testing

andyavt committed:
mistakes fixing

andyavt committed:
fairwaves architecture_markdown

andyavt committed:
Added Fairwaves architecture to the main doc

andyavt committed:
Picture

andyavt committed:
weird signs fix

andyavt committed:
markups after proofreading

andyavt committed:
markups publishing

andyavt committed:
Power supplies + Schema

andyavt committed:
CMD57 testing procedure

andyavt committed:
link fixing

andyavt committed:
umtrx html testing

shamrin commented on markdown/README.markdown:
```diff
@@ -0,0 +1,234 @@
+Hacker's guide to UmDESK
+========================
```
Oh, my draft Hacker's guide is here! I realize you don't use this text anymore, but I feel funny reading it now  :smiley: Please keep it in the repo!

andyavt committed:
osmo-trx rebuild correction

andyavt committed:
Branch release set update

-Branch release set update
-Statistics added

andyavt committed:
Page proof of the last commit

andyavt committed:
final step)

andyavt committed:
Statistics added to main page

andyavt committed:
Header location issue fixed

andyavt committed:
Statistics

andyavt committed:
Jitter Buffer added

andyavt committed:
serial and tcxo added to procedure

andyavt committed:
html ver of procedure updated

andyavt committed:
2016

andyavt committed:
Dropdown menu update& manuals update

- Dropdown menu for UmSite,Umdesk, Software conf
- Manuals have updated  according to updated software

andyavt committed:
Network configuration

andyavt committed:
new_FAQ_draft

andyavt committed:
Recent questions from sales

DmitryKlyuykov committed:
Added manual about OpenBTS-UMTS installation with UmTRX

3G_m.png - preview picture
openbts-umts.html - article
index.html - added preview

DmitryKlyuykov committed:
fixed non-UTF-8 minus sign

DmitryKlyuykov committed:
fixed quick start case and picture for 3G

andyavt committed:
Style Fix

- Added new Grid
- Fixed header
- Menu

DmitryKlyuykov committed:
Coverage and capacity document added

DmitryKlyuykov committed:
coverage and capacity pictures

DmitryKlyuykov committed:
typos fixed

DmitryKlyuykov committed:
more typo fixed

DmitryKlyuykov committed:
dot removed from article description

DmitryKlyuykov committed:
Updated with encryption configuration

andyavt committed:
umtrx_query_sensors added to UmSite manual

andyavt committed:
Real time output power control updates

andyavt committed:
Real time output power control little fix

andyavt committed:
Real time output power control syntaxis link fix

andyavt committed:
adding local pictures

andyavt committed:
Delete content.zip

andyavt committed:
Replacing hosted images to local

andyavt committed:
architecture images replacing

andyavt committed:
images replacing
